exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/project/RequestedTupleImpl.java

```diff
@@ -74,6 +75,7 @@
 public class RequestedTupleImpl implements RequestedTuple {
 
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(RequestedTupleImpl.class);
+  private static final Collection<SchemaPath> PROJECT_ALL = ImmutableList.of(SchemaPath.STAR_COLUMN);
```

Comment:
Consider using build-in Java methods instead of guava when possible: `Collections.sigletonList`.